cmd/neofs-ir/defaults.go

```diff
@@ -64,6 +64,7 @@ func defaultConfiguration(cfg *viper.Viper) {
 	cfg.SetDefault("contracts.container", "")
 	cfg.SetDefault("contracts.audit", "")
 	// alphabet contracts
+	cfg.SetDefault("contracts.alphabet.amount", 7)
```

Comment:
@alexvanin, I don't think we need default values at all, since they all are empty strings. About dev-env and spec: can you create some note to do it after the release?

Reply:
> I don't think we need default values at all, since they all are empty strings. 

`defaults.go` keeps the list of all configurable parameters of the application. If alphabet default amount is `7`, then let's provide seven valid configuration keys in this file.

> About dev-env and spec: can you create some note to do it after the release?

Done: https://github.com/nspcc-dev/neofs-dev-env/issues/50 and https://github.com/nspcc-dev/neofs-spec/issues/26

Reply:
It does not keep the list, it sets the default values, so the user knows nothing about the actual list of the configurable parameters. As a developer tip, I can list all possible glagolitic letters, but I'm not sure we need it at all, because there is no default smart-contract addresses.

Reply:
> I can list all possible glagolitic letters, but I'm not sure we need it at all

I am sure we need it, but 40+ letters are too much. Therefore I suggest to keep it only seven of them as default value for `contracts.alphabet.amount` is seven.

Otherwise remove alphabet contract mentions from defaults.go

Reply:
I removed addresses alphabetic contracts from config defaults.